### PR TITLE
10.0 sale_comment_template

### PR DESCRIPTION
**Pull request by SimoRubi**
Hi,
I've fixed [this PR](https://github.com/OCA/sale-reporting/pull/40) for migration to 10.0 of the module sale_comment_template.
Please check

### CODE REVIEW

File: sale_comment_template/models/__init__.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+#
+#
+# Copyright Camptocamp SA
+# author nbessi
+#
+#
```
Comment (yajo) [inline sale_comment_template/models/__init__.py]: Use this template for init files: https://github.com/OCA/maintainer-tools/blob/c73e7d8ca936cce206256935ec20ad7083b7f176/template/module/__init__.py#L1-L2

File: sale_comment_template/__manifest__.py [hunk 2]
```diff
@@ -20,16 +20,17 @@
 #
```
Comment (yajo) [inline sale_comment_template/__manifest__.py]: Use this template for non-init files: https://github.com/OCA/maintainer-tools/blob/c73e7d8ca936cce206256935ec20ad7083b7f176/template/module/__openerp__.py#L1-L3

File: sale_comment_template/models/sale_order.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
```
Comment (yajo) [inline sale_comment_template/models/sale_order.py]: Update header

**Review by yajo — CHANGES_REQUESTED** [review]

**Comment by yajo** [comment]
Bots fail

**Comment by SimoRubi** [comment]
@Yajo thanks for the review, bots fail because this module depends on two modules in [account-invoice-reporting](https://github.com/OCA/account-invoice-reporting), and here we haven't an oca_dependencies file.

**Comment by yajo** [comment]
Then add it

**Comment by SimoRubi** [comment]
depends on https://github.com/OCA/account-invoice-reporting/pull/65

**Comment by tafaRU** [comment]
@Yajo, please also have a review of its dependency https://github.com/OCA/account-invoice-reporting/pull/65
Thank you in advance.

File: sale_comment_template/tests/__init__.py [hunk 4]
```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from . import test_sale_order_report
```
Comment (cubells) [inline sale_comment_template/tests/__init__.py]: A blank line is needed

**Review by cubells — APPROVED** [review]
👍 

Please, remove sale_comment_template.pot file. It's not needed.

MInor changes.

LGTM

Code review and tests.

File: sale_comment_template/tests/__init__.py (hunk 4 above)
Comment (SimoRubi) [reply sale_comment_template/tests/__init__.py]: Done

Comment (cubells) [reply sale_comment_template/tests/__init__.py]: Thanks @SimoRubi 

@Yajo can you retest?

**Review by tafaRU — APPROVED** [review]

**Comment by tafaRU** [comment]
@Yajo, this PR seems mergeable, could you please check it?

File: sale_comment_template/README.rst [hunk 5]
```diff
@@ -12,17 +17,41 @@ Two positions are available for the comments:
 
 Texts are copied on the invoice when you will create invoice from sale order.
 
-Dependencies
-------------
+Usage
+=====
 
-The modules `invoice_comment_template` and `base_comment_template` are
```
Comment (pedrobaeza) [inline sale_comment_template/README.rst]: There should be a section `Installation` similar to this one, indicating that this module depends on `invoice_comment_template` and `base_comment_template` in OCA/account-invoice-reporting.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: sale_comment_template/README.rst (hunk 5 above)
Comment (SimoRubi) [reply sale_comment_template/README.rst]: Added the <code>Installation</code> section

**Review by yajo — APPROVED** [review]

**Comment by yajo** [comment]
@pedrobaeza OK with this?

**Comment by tafaRU** [comment]
@pedrobaeza, ping!
Could you please have a final review?

**Review by pedrobaeza — APPROVED** [review]

**Comment by tafaRU** [comment]
Thank you, @pedrobaeza.

**Pull request merged**